README.md

```diff
@@ -100,3 +100,38 @@ $cat shibboleth/overrides/etc/shibboleth/metadata-providers.xml
 </MetadataProviders>
 ```
 configures 2 metadata sources
+
+# Case study - local testing
+
+1. create .env
+   ```
+   SHIB_HOSTNAME=jm
+   NGINX_TOMCAT=dev-5.pc:85
+   NGINX_NODE=dev-5.pc:85
+   NGINX_SERVER_NAMES="jm localhost"
+   NGINX_RESOLVERS="1.1.1.1 8.8.8.8"
+   NGINX_MAX_BODY_SIZE=10G
+   ```
+2. in ./nginx/ssl
+   ```
+   openssl dhparam -out dhparam.pem 4096
+   openssl genrsa -out /out/serverkey.pem 2048
+   openssl req -new -x509 -key /out/serverkey.pem -out /out/servercert.pem -days 365
+   cp ./servercert.pem ./nginx_chain_cert.pem
+   ```
+   (on win, you can use `docker run -it --rm  -v %cd%:/out nginx /bin/bash -c "CMD"`)
+3. in ./
+   ```
+   mkdir -p ./shibboleth/overrides/etc/shibboleth/
+   curl https://www.eduid.cz/docs/metadata2021.eduid.cz.crt.pem -o ./shibboleth/overrides/etc/shibboleth/metadata2021.eduid.cz.crt.pem
+   ```
+3. in ./
+   ```
+   docker compose up
+   ```
+5. open https://jm/Shibboleth.sso/DiscoFeed
+6. optionally, you can inspect logs, unless you changed the default contents or created `proxied-idp.xml`, there will be errors/warnings in the log
+   ```
+   docker exec -it dockerized-nginx-with-shibboleth-shibboleth-1 /bin/bash -c "tail /opt/shibboleth-sp/var/log/shibboleth/shibd_warn.log -f"
```

Comment:
```
docker compose exec shibboleth tail -f /opt/shibboleth-sp/var/log/shibboleth/shibd_warn.log
```
should also work, or mount the logs volume and inspect in any container
```
docker run -v dockerized-nginx-with-shibboleth_logs:/logs --rm debian:stable-slim ls -laR /logs/
/logs/:
total 28
drwxr-xr-x 4 root root  4 Jan 11 21:16 .
drwxr-xr-x 1 root root  3 Feb 20 14:52 ..
drwxr-xr-x 2 root root  6 Feb  1 00:29 nginx
drwxr-xr-x 2 root root 12 Feb 19 23:43 shibboleth

/logs/nginx:
total 4139
drwxr-xr-x 2 root root        6 Feb  1 00:29 .
drwxr-xr-x 4 root root        4 Jan 11 21:16 ..
-rw-r--r-- 1 root root 15032577 Jan 31 23:37 access.log-2024-01
-rw-r--r-- 1 root root 13561102 Feb 20 14:20 access.log-2024-02
-rw-r--r-- 1 root root   791572 Jan 31 23:14 error.log-2024-01
-rw-r--r-- 1 root root   658855 Feb 20 14:19 error.log-2024-02

/logs/shibboleth:
total 1739
drwxr-xr-x 2 root root      12 Feb 19 23:43 .
drwxr-xr-x 4 root root       4 Jan 11 21:16 ..
-rw-r--r-- 1 root root  576322 Feb 20 14:40 shibd.log
-rw-r--r-- 1 root root 1000021 Feb 19 23:43 shibd.log.1
-rw-r--r-- 1 root root 1000029 Feb 12 23:43 shibd.log.2
-rw-r--r-- 1 root root 1000053 Feb  7 08:35 shibd.log.3
-rw-r--r-- 1 root root 1000074 Jan 28 23:33 shibd.log.4
-rw-r--r-- 1 root root 1006712 Jan 18 14:32 shibd.log.5
-rw-r--r-- 1 root root 1000091 Jan 14 00:06 shibd.log.6
-rw-r--r-- 1 root root    5993 Feb  5 11:33 shibd_warn.log
-rw-r--r-- 1 root root       0 Jan 11 21:06 signature.log
-rw-r--r-- 1 root root   83365 Feb 20 13:58 transaction.log
```

Reply:
agree, there are a lot of ways how to do this, if you want yours to be in this PR, please update the PR

Reply:
nah, just wanted to make sure it's clear how the logs are stored